packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestDocCommon.scss

```diff
@@ -0,0 +1,171 @@
+// stylelint-disable scss/at-mixin-pattern
+
+@mixin hookshot-common-wrapper() {
+  border: 1px dashed #808080;
+  height: 145px;
+  width: 540px;
+}
+
+@mixin hookshot-common-content() {
+  background-color: #d3d3d3;
+  height: 36px;
+  margin-left: 220px;
+  margin-right: 220px;
+  margin-top: 55px;
+  width: 100px;
+}
+
+:local {
+  .default-hookshot-wrapper {
+    height: 40px;
+    width: 200px;
+  }
+
+  .content-wrapper {
+    border: 1px dashed #808080;
+    height: 145px;
+    position: relative;
+    width: 500px;
+  }
+
+  .content-wrapper button {
+    left: 210px;
+    position: absolute;
+    top: 50%;
+  }
+
+  .hookshot-wrapper {
+    border: 1px dashed #808080;
+    height: 300px;
+    width: 300px; 
+  }
+
+  .hookshot-content-wrapper {
+    background-color: #b0e0e6;
+    height: 36px;
+    width: 200px;
+  }
+
+  .hookshot-primary {
+    @include hookshot-common-content();
+  }
+
+  .hookshot-auto-flipped {
+    @include hookshot-common-content();
+    margin-left: 50px;
+  }
+
+  .hookshot-auto-90 {
+    @include hookshot-common-content();
+    margin-left: 60px;
+  }
+
+  .hookshot-auto--90 {
```

Comment:
can we change this class name to something more meaningful? `hookshot-auto-90` and `hookshot-auto--90` don't have any differentiation

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.